packages/reports/package.json

```diff
@@ -74,7 +74,7 @@
     "ember-resize": "0.0.17",
     "ember-resolver": "^4.0.0",
     "ember-route-action-helper": "^2.0.6",
-    "ember-sortable": "^1.10.0",
+    "ember-sortable": "^1.11.2",
```

Comment:
Are we using this for the series selector?

Reply:
I was trying to fix the reorder test helper.  did not work.